pkg/kubectl/cmd/expose.go

```diff
@@ -212,19 +199,19 @@ func RunExpose(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []str
 	}
 	// TODO: extract this flag to a central location, when such a location exists.
 	if cmdutil.GetFlagBool(cmd, "dry-run") {
-		fmt.Fprintln(out, "running in dry-run mode...")
-	} else {
-		data, err := info.Mapping.Codec.Encode(object)
-		if err != nil {
-			return err
-		}
-		object, err = resource.NewHelper(info.Client, info.Mapping).Create(namespace, false, data)
-		if err != nil {
-			return err
-		}
+		fmt.Fprintln(out, dryRunMessage)
```

Comment:
This prevents piping.  I'd like to see it removed.  Is there a strong reason to keep it?